frontend/src/components/crop/SelectionOperations.ts

```diff
@@ -101,8 +143,12 @@ export default function defineSelectionOperations(
       );
     },
     removeAll: function () {
-      removeAll(imageSelectionsCopy, setSelections);
+      removeAll(imageSelectionsCopy, setSelections, minSavePrevious);
+    },
+    toPrevious: function () {
+      toPrevious(previousImageSelections, setSelections, setPreviousSelections);
     },
+    undo_possible: previousImageSelections.length != 0,
```

Comment:
Snake case to camel case, and non-strict to strict inequality operator: 
```suggestion
    undoPossible: previousImageSelections.length !== 0,
```